app/src/Users.vue

```diff
@@ -17,13 +17,14 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
 import axios from 'axios'
 import AddUserModal from './components/users/AddUserModal.vue'
 import UserTable from './components/users/UserTable.vue'
 import Button from './components/common/Button.vue'
 import Component from 'vue-class-component'
 import Vue from 'vue'
+import UserModel from "@/models/UserModel";
```

Comment:
```suggestion
import UserModel from '@/models/UserModel'
```

Reply:
Thanks, just fixed it in the most recent commit!